tests/integration/targets/azure_rm_virtualmachine/tasks/azure_test_skus.yml

```diff
@@ -0,0 +1,137 @@
+- name: Set variables
+  ansible.builtin.include_tasks: setup.yml
+
+- name: Gather Resource Group info
+  azure.azcollection.azure_rm_resourcegroup_info:
+    name: "{{ resource_group }}"
+  register: rg_info
+
+- name: List available VM SKUs
+  azure.azcollection.azure_rm_vmskus:
+    location: "{{ rg_info.resourcegroups.0.location }}"
+    resource_type: "virtualMachines"
+    size: "standard_B1"
+    zone: True
+  register: available_skus_result
+
+- name: Create desired capabilities list
+  ansible.builtin.set_fact:
+    desired_capabilities: [
+        {
```

Comment:
Wrong indentation: expected 6 but found 8,  Change it the same way below

Reply:
Fixed